notebooks/214-grammar-correction/214-grammar-correction.ipynb

```diff
@@ -27,23 +27,29 @@
     "- Install prerequisites\n",
```

Comment:
<p>why only decoder with past?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Because only decoder with past is quantized</p>